INSTALL.md

```diff
@@ -178,66 +240,42 @@ display_errors = Off
 upload_tmp_dir is set to a correct default value that will work on your system
 error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT & ~E_NOTICE
 ```
-Make sure that settings in MYSQL /etc/mysql/my.cnf file include:
+- Make sure that settings in MySQL/etc/mysql/my.cnf file include:
```

Comment:
Why did you remove the space here?

Reply:
It wasn't consistent with the rest of the document. This is what it looks like now.
![image](https://user-images.githubusercontent.com/41968151/48683532-51be6a00-eb73-11e8-84d3-d6202b18d973.png)

Reply:
You can't do that. That is a file path.

Reply:
Oh you are talking about that space. The file path shouldn't have a space in it anyway though right?